packages/katello/katello-host-tools/katello-host-tools.spec

```diff
@@ -70,10 +70,16 @@ Group:      Development/Languages
 
 Conflicts: pulp-consumer-client
 
+%if %{dnf_install}
+Requires: gofer >= 2.12
+%else
 %if 0%{?rhel} == 5
 Requires: gofer >= 2.5
+Requires: gofer < 2.11
```

Comment:
Does it actually not work with `2.11` or higher?

Reply:
Untested, but I simply don't envision moving el5 clients from 2.7 to 2.11 do you?